x11rb-protocol/src/parse_display/mod.rs

```diff
@@ -38,20 +38,41 @@ impl ParsedDisplay {
 /// Parse an X11 display string.
 ///
 /// If `dpy_name` is `None`, the display is parsed from the environment variable `DISPLAY`.
+///
+/// This function is only available when the `std` feature is enabled.
+#[cfg(feature = "std")]
 pub fn parse_display(dpy_name: Option<&str>) -> Option<ParsedDisplay> {
+    fn file_exists(path: &str) -> bool {
+        let path: &std::path::Path = path.as_ref();
+        path.exists()
```

Comment:
Can you use `std::path::Path::new(path).exists()`?

Reply:
Argh. Thanks for reading the docs a lot better than I did! Done in #860